### DISCUSSION TITLE
Commenting generated structures

### DISCUSSION THREAD

**Discussion by 4ydx** [sqlc-dev/sqlc#3151] [Feature Requests & Ideas]
Would it be possible to provide a way to add comments to the structs that are generated?  That is, comments for individual struct members (db columns)?  Maybe something that pulls postgresql comments for instance.

**Comment by 4ydx** [comment]
I see now that [postgres comments ](https://www.postgresql.org/docs/current/sql-comment.html) are added to the models.go structs.  Would it be possible to add them to the generated Row structs too?

**Comment by ajwerner** [comment]
I'm also interested in this. In particular, we're using SQLite. Before we adopted migrations, it was okay because the schema was all in one place. Now that we use migration files, it's hard to keep track of commentary on what a column means.
